Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -458,6 +478,26 @@ extension MapboxMap: MapEventsObservable {
     }
 }
 
+// MARK: - Map data clearing -
+
+extension MapboxMap {
```

Comment:
fyi this is not an extension function, but a function of `MapboxMap`. But I guess this can be just an implementation detail.

Reply:
It's just an implementation detail and won't impact public API.